crates/iota-proxy/src/peers.rs

```diff
@@ -115,13 +115,10 @@ impl IotaNodeProvider {
     pub fn get_mut(&mut self) -> &mut IotaPeers {
         &mut self.active_validator_nodes
     }
-    fn update_active_validator_set(&self, summary: &IotaSystemStateSummary) {
+    fn update_committee_member_set(&self, summary: &IotaSystemStateSummary) {
```

Comment:
Should `IotaNodeProvider::active_validator_nodes` also assume the role of "committee members"? Or should `IotaNodeProvider` be extended with `committee_members` and the corresponding logic?